LayoutTests/media/encrypted-media/clearKey/clearKey-cenc-stop-playback-before-key.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Playback ClearKey CENC audio with full-sample encryption</title>
+    <script src="../medias-enc.js"></script>
+    <script src="../../video-test.js"></script>
+    <script src="../../media-source/media-source-loader-simple.js"></script>
+    <script src="encrypted-media-clearKey-handler.js"></script>
+    <script>
+
+     const audioConf = streamMedias["simpleClearKeyMSE"].audio;
+
+     function runTest() {
+
+         findMediaElement();
+
+         const emeHandler = new EncryptedMediaHandler(video, audioConf);
+         if (!emeHandler)
+             failTest();
+
+         emeHandler.skipUpdateCallback = function() {
+             setTimeout(function() {
+                 logResult(true, "Changing URL");
+                 const startTime = Date.now();
+                 video.src = "../../content/counting.mp4"
+                 video.play(1);
+                 const difference = Date.now() - startTime;
```

Comment:
Found 1 new test failure: media/encrypted-media/clearKey/clearKey-cenc-stop-playback-before-key.html

Reply:
Yes, it's because moving handling the state change after calling the superclass. That does not break the wait. Added a comment about it. Also added skip for mac.